manuscript.md

```diff
@@ -36,6 +36,40 @@
 
 ## What is a species interaction?
 
+Species interaction encompass every action occuring between any set
+of two species (or more?) which can be declined into two main types: 
+positive and negative interactions (also neutral?) [@Morales-Castilla2015InfBio].
+As listed by [@Jordano2016ChaEco], these types of interaction may take many
+forms, where positive interactions could for example be mutualism, 
+commensalism and symbiosis and negative interactions be predation, herbivory
+and parasitism.
+
+Despite the multitude type/form of interactions,
+network ecologists have primarily focused their research on food webs 
+[@Morales-Castilla2015InfBio, @Dunne2006NetStr, ...] when considering 
+interactions across a community or pairwise interactions such as plant-pollinator
+and host-parasite interactions. Arguably the main reason for the bulk of research 
+focusing on these subsets of interaction types may, in part, be due to the difficulty
+in accurately describing or observing other types of interactions and may be underpinning
+a scarcity of data. However, a *mechanistic* understanding of what determines if
+and how species will interact can be leveraged when trying to predict interactions 
+when data are incomplete or missing.
+
+The representation of species interactions or interaction networks in the
+form of graphs and matrices is quite effective for their description and analysis.
+It allows the calculation of many network properties (do we list some?), which
+some can even be use to help the prediction of further interactions,
+by for example applying a constraint on the possible number of links based on
+the number of species present [@MacDonald2020RevLin].
+
+In a graph, each species is represented as a node and each interaction as an edge
+[@Delmas2018AnaEco, @Pascual2006EcoNet], whereas two nodes linked by
+and edge represents a species interaction. When the matrix format is used,
+each row represents a consumer species and each column represents a 
+ressource species. For a presence/absence interaction network, the 
```

Comment:
We can probably build on this since the values won't always be binary and this would tie back that we could have +ve or -ve values and then when stepping into the work of weighed interactions this could take on a range of values...